src/modules/nodejs.rs

```diff
@@ -56,19 +57,29 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
             })
             .map(|variable| match variable {
                 "version" => {
-                    let version = nodejs_version
+                    let node_ver = nodejs_version
                         .deref()
                         .as_ref()?
                         .trim_start_matches('v')
                         .trim();
 
                     VersionFormatter::format_module_version(
                         module.get_name(),
-                        version,
+                        node_ver,
                         config.version_format,
                     )
                     .map(Ok)
                 }
+                "engines_version" => {
+                    let node_ver = nodejs_version.as_deref();
+                    let eng_ver = get_engines_version(context);
+                    let in_engines_range = check_engines_version(node_ver, eng_ver.clone());
+
+                    match (eng_ver, in_engines_range) {
+                        (Some(ver), false) => Some(Ok(ver)),
+                        _ => None,
+                    }
```

Comment:
On second look, this would look a bit better:
```suggestion
                    let eng_ver = get_engines_version(context)?;
                    let in_engines_range = check_engines_version(node_ver, eng_ver.as_deref());

                    (!in_engines_range).then_some(Ok(ver))
```

To replace the `clone` with `as_deref` please change the signature to `Option<&str>`.

Reply:
Implemented this!